templates/download/desktop/index.html

```diff
@@ -53,7 +53,7 @@ <h2>Easy ways to switch to Ubuntu</h2>
             <h3 class="p-heading-icon__title">From an older version</h3>
           </div>
         </div>
-        <p class="p-card__content">If you&rsquo;re already running Ubuntu, you can upgrade in a few clicks from <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/tutorial-upgrading-ubuntu-desktop">Software Updater</a>.</p>
+        <p class="p-card__content u-no-margin--top">If you&rsquo;re already running Ubuntu, you can upgrade in a few clicks from <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/tutorial-upgrading-ubuntu-desktop">Software Updater</a>.</p>
```

Comment:
All `.p-card__content` does is push the text closer to the title, so you could probably get away with just not having any classes attached to these paragraphs